### DISCUSSION TITLE
Remix's React 18 support?

### DISCUSSION THREAD

**Discussion by pagebanc** [remix-run/remix#2953] [Q&A] [answered]
Hello,
Much anticipated React 18 has been released for 20+ days.
As of today (4/21/2022), `npx create-remix@latest` still creates the application structure that is React 17.
I know there are pull requests related to this (e.g., https://github.com/remix-run/remix/issues/2564) but...
Does anyone have the info about the **timeline for React 18 support**?
Thank you!

**Comment by sergiodxa** [comment, marked as answer]
Templates are not updated but if you upgrade you can use React 18 already

**Reply by pagebanc** [reply]
Thank you @sergiodxa
So you mean upgrade the following to 18.0.0 in package.json?
```
"react": "^17.0.2",
"react-dom": "^17.0.2"
```
Is that it?

**Reply by sergiodxa** [reply]
Yes, and if you use TS the types for react and react-dom

**Reply by pagebanc** [reply]
Okay.
Thanks again.
